### PR TITLE
cs cleanup

### PR DESCRIPTION
**Pull request by digitalkaoz**
it seems nobody is checking the CS before creating a PR ;)

maybe add https://styleci.io/ webhook to PRs ?

### CODE REVIEW

File: src/Composer/IO/BaseIO.php [hunk 1]
```diff
@@ -121,8 +121,8 @@ public function loadConfiguration(Config $config)
     /**
      * System is unusable.
      *
-     * @param string $message
-     * @param array $context
+     * @param  string $message
```
Comment (kamilsk) [inline src/Composer/IO/BaseIO.php]: here is different https://github.com/composer/composer/pull/5327/files#diff-b0c94b96dc86be178b130cd557412587R713

Comment (digitalkaoz) [inline src/Composer/IO/BaseIO.php]: mh the docblocks should be aligned...whats wrong?

**Comment by staabm** [comment]
@digitalkaoz Seldaek will update/fix the CS from time to time. PRs to the repos will land even if they not stick to the CS.

File: src/Composer/IO/BaseIO.php (hunk 1 above)
Comment (kamilsk) [inline src/Composer/IO/BaseIO.php]: `@param string` vs `@param  string`, one space vs two space. why?

**Comment by digitalkaoz** [comment]
@staabm ok, ill close it then. thanks for the info! and yes its a good practise :)

**Pull request closed without merging**